stubs/ServiceDocumentRepository.stub

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Doctrine\Bundle\MongoDBBundle\Repository;
+
+use Doctrine\ODM\MongoDB\Repository\DocumentRepository;
+
+/**
+ * @template TDocumentClass
```

Comment:
I think it has to be `TDOcumentClass of object` to work correctly. See: https://github.com/phpstan/phpstan-doctrine/blob/4cd02538b9ae220878f59b5c26b873adc1126ec9/stubs/DocumentRepository.stub#L8